pkg/azuredisk/nodeserver.go

```diff
@@ -494,15 +568,36 @@ func getNodePublishMountOptions(req *csi.NodePublishVolumeRequest) []string {
 	return mountOptions
 }
 
-func (d *Driver) getBlockSizeBytes(devicePath string) (int64, error) {
-	output, err := d.mounter.Exec.Run("blockdev", "--getsize64", devicePath)
+func statFS(path string) (available, capacity, used, inodesFree, inodes, inodesUsed int64, err error) {
```

Comment:
Those metrics have already been implemented in k/k. Let's reuse existing codes, e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/metrics_statfs.go

Reply:
Okay, I'll make the changes.